torch/_torch_docs.py

```diff
@@ -8012,6 +8012,8 @@ def merge_dicts(*dicts):
 If :attr:`compute_uv` is ``False``, the returned `U` and `V` matrices will be zero matrices
 of shape :math:`(m \times m)` and :math:`(n \times n)` respectively. :attr:`some` will be ignored here.
 
+Supports real-valued and complex-valued input.
```

Comment:
Future work: specify the dtypes explicitly since "real-valued" would be every non-bool non-complex dtype